workers/questionreviewer/Dockerfile

```
@@ -0,0 +1,27 @@
+
+RUN apk add --no-cache git
+
+COPY go.mod go.sum ./
+
+RUN go mod download
+
+COPY . .
+
+RUN CGO_ENABLED=0 GOOS=linux go build -o questionreviewer ./main.go
+
+FROM alpine:latest AS runtime
```

Hadolint (2.12.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)